vor/test/test_elasticsearch.py

```diff
@@ -1490,17 +1493,20 @@ def setUp(self):
     },
     "cluster_name" : "production"
 }"""
-        data = simplejson.loads(stats)
-        self.collector.flatten(data)
+        self.collector.flatten(simplejson.loads(stats))
```

Comment:
It's either call it twice, or call it once and then make a deepcopy of the loaded data because `flatten` modifies the data in place. Seemed simple to just inline load it, but feel free to change to a copy if you like it better.